packages/okidoc-md/src/utils/nodeAST.js

```diff
@@ -21,6 +22,39 @@ function cleanUpFunction(node, { identifierName, JSDocCommentValue } = {}) {
   node.decorators = [];
   node.body = t.blockStatement([]);
 
+  if (node.typeParameters) {
+    /*
+      Removes generic constraint, because they can't be parsed by documentation.js
+      `function<T extends string>(param: T): T`
+     */
```

Comment:
It may be helpful to print a warning message to the console here, saying that generic constraints are ignored. What do you think?

Reply:
`documentation.js` doesn't print info about generic type to the output docs. I don't want to add some console logs about issues that a user can't fix.